src/Libraries/CoreNodeModels/ColorRange.cs

```diff
@@ -32,6 +32,15 @@ namespace CoreNodeModels
     [AlsoKnownAs("DSCoreNodesUI.ColorRange")]
     public class ColorRange : NodeModel
     {
+        private List<Color> _defaultColors;
+        private List<Color> DefaultColors => _defaultColors ??= DefaultColorRanges.Analysis.ToList();
```

Comment:
it does not look like you use any list functionality here, is there a reason you use this type here and in the create helpers?

Reply:
CreateParametersForColors() was already accepting List<Color> and didn't want to chage existing code unless I really have to. Hepe the latest commit is okay.